core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockHttpGoogleBigQueryDAO.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import akka.actor.ActorSystem
+import org.broadinstitute.dsde.workbench.google.HttpGoogleBigQueryDAO
+import org.broadinstitute.dsde.workbench.google.GoogleCredentialModes.GoogleCredentialMode
+
+import scala.concurrent.ExecutionContext
+
+class MockHttpGoogleBigQueryDAO(appName: String,
+                                googleCredentialMode: GoogleCredentialMode,
+                                workbenchMetricBaseName: String)
+                               (implicit system: ActorSystem, executionContext: ExecutionContext)
+  extends HttpGoogleBigQueryDAO(appName, googleCredentialMode, workbenchMetricBaseName)
```

Comment:
this should override the relevant methods so it doesn't try to call Google (unless you really want this to call Google)